DependencyInjection/Compiler/SerializerConfigurationPass.php

```diff
@@ -41,18 +41,18 @@ public function process(ContainerBuilder $container)
             throw new \InvalidArgumentException('Neither a service called "jms_serializer.serializer" nor "serializer" is available and no serializer is explicitly configured. You must either enable the JMSSerializerBundle, enable the FrameworkBundle serializer or configure a custom serializer.');
         }
 
-        if ($container->has('serializer')) {
+        if ($container->has('serializer') || $container->has('jms_serializer.serializer')) {
```

Comment:
I think it would be better to add a case before the others instead (move the one line 65).